KyryloRomanenko/HW5/Task 1. loop.py

```diff
@@ -0,0 +1,19 @@
+item = 0
```

Comment:
look at this code

list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)